src/include/OSL/oslexec.h

```diff
@@ -137,10 +137,11 @@ class OSLEXECPUBLIC ShadingSystem
     ///    int no_noise           Replace noise with constant value. (0)
     ///    int no_pointcloud      Skip pointcloud lookups. (0)
     ///    int exec_repeat        How many times to run each group (1).
-    ///    int opt_warnings       Warn on certain failure to runtime-optimize
-    ///                              certain shader constructs. (0)
-    ///    int gpu_opt_error      Consider a hard error if certain shader
-    ///                              constructs cannot be optimized away. (0)
+    ///    int opt_warnings       Warn on failure to runtime-optimize certain
+    ///                              shader constructs. (0)
+    ///    int gpu_opt_error      Issue a hard error if certain shader
+    ///                              constructs cannot be optimized away, which
```

Comment:
Can we turn on gpu_opt_error even if we are running on the cpu?  (In order to get a shader into shape before trying the GPU)

Reply:
That is 100% the point, yes.